doc/classes/Generic6DOFJoint.xml

```diff
@@ -4,91 +4,119 @@
 		The generic 6-degrees-of-freedom joint can implement a variety of joint types by locking certain axes' rotation or translation.
 	</brief_description>
 	<description>
-		The first 3 DOF axes are linear axes, which represent translation of Bodies, and the latter 3 DOF axes represent the angular motion. Each axis can be either locked, or limited.
+		The first 3 DOF axes are linear axes, which represent translation of Bodies, and the latter 3 DOF axes represent the angular motion. Each axis can be either locked, or limited. The joint may also take on spring and motor characteristics, influencing its linear and angular movement.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="get_use_global_rotation" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Bingus.
```

Comment:
Hmm.

Reply:
> @19PHOBOSS98 The more you push to this PR, the more Godot's CI runs. Godot has limited CI resources and this PR is eating them up. I'm closing this, we can re-open once your PR is nearer to completion. Based on your most recent push I can only assume that the PR still has a long ways to go before it's ready to be reviewed and merged.

sorry about that...

Reply:
> Hmm.

yeah, was trying to figure out why my method wasn't showing up in the docs view... I already bound it to ClassDB, updated the xml files by running --doctool. My new properties carried over but my methods didn't show...

The functions works, I can call them from GDScript without fail.

![image](https://user-images.githubusercontent.com/37253663/188295062-3deecbc1-6237-4e39-adbb-61f455b07b44.png)

![image](https://user-images.githubusercontent.com/37253663/188295064-6c73b567-aff8-408c-89b3-3e4b599ed5ca.png)

Do you have any advise? I've been everywhere from RocketChat, to Reddit, to Discord... help would be much appreciated

Reply:
> Hmm.

I figured that the CI could tell me what's wrong but then there was a referencing issue so I tried to fix that and yeah...

Reply:
The CI will also run on your repo, so you don't need to use Godot's CI resources to run CI. Just run on a branch that isn't opened as a PR. https://github.com/19PHOBOSS98/godot/actions

Also, you really should not open a PR from one of your main branches (like `master` or `3.x`), you should create named branches for your PRs.

Reply:
thanks, you have anything for the first problem?

Reply:
@19PHOBOSS98 They are showing up when doctool is run.

The method `get_use_global_rotation` is documented here:

```xml
<member name="angular_spring/use_global_rotation" type="bool" setter="set_use_global_rotation" getter="get_use_global_rotation" default="false">
...
</member>
```

Reply:
Your PR also has a lot of formatting errors. You can fix most by using these scripts: https://github.com/godotengine/godot/tree/master/misc/scripts (if you're on Windows, to run `.sh`, use [WSL](https://apps.microsoft.com/store/detail/ubuntu/9PDXGNCFSCZV)).

Reply:
> @19PHOBOSS98 They are showing up when doctool is run.
> 
> The method `get_use_global_rotation` is documented here as a part of the property:
> 
> ```
> <member name="angular_spring/use_global_rotation" type="bool" setter="set_use_global_rotation" getter="get_use_global_rotation" default="false">
> ...
> </member>
> ```
> 
> The methods in your second screenshot are not trivial getters/setters for member properties.

I'm sorry I think I wasn't clear enough.
![image](https://user-images.githubusercontent.com/37253663/188299930-9f06f387-9a8e-4530-b7c6-807e4d80668e.png)
I'm aware that `get_use_global_rotation()` along with my other methods show up in the properties section but somehow they don't show up in the methods section. The methods in my second screenshot were already there before I started my PR:
![image](https://user-images.githubusercontent.com/37253663/188300114-9cf4f0ac-2de6-455b-b8c4-9c862e74e63c.png)

![image](https://user-images.githubusercontent.com/37253663/188300105-f412e742-be57-4b10-92ea-85cc42399e82.png)

Trivial or not, they show up in the docs view... but mine don't. Why?

Reply:
> Your PR also has a lot of formatting errors. You can fix most by using these scripts: https://github.com/godotengine/godot/tree/master/misc/scripts (if you're on Windows, to run `.sh`, use [WSL](https://apps.microsoft.com/store/detail/ubuntu/9PDXGNCFSCZV)).

thank you!

Reply:
@19PHOBOSS98 The methods in your screenshot are not trivial getters/setters for properties. A trivial getter/setter for a property has a get method that returns a value and takes zero arguments, and a set method that is void and takes one argument. The methods you linked have getters that take one argument and setters that take two arguments.

Reply:
I think I get it. Basically setters and getters for a specific property are meant to **not** show up in the methods section and instead are meant to hangout with their property over at the properties section. The methods I saw threw me off I guess. They were a bit weird for having that many arguments, right, right.

I got to credit [kleonc](https://www.reddit.com/user/kleonc/) too over at reddit:
https://www.reddit.com/r/godot/comments/x5amba/i_need_help_in_class_documentation_new_methods/in219n3/?context=3

@aaronfranke welp, I think I'm done with my changes and it looks like it passed all the checks:
![image](https://user-images.githubusercontent.com/37253663/188319366-ce2b0315-2a8b-4d8e-8c32-0abd791cbe89.png)

you can re-open the PR now, I'm done adding on top of this. Sorry again for the trouble.